index.js

```diff
@@ -68,7 +68,37 @@ function toggleColor(element) {
   }
 }
 
-/***** Deliverable 2 *****/
+const header = document.querySelector("#header")
+
+header.addEventListener("click", function(){
+  toggleColor(header)
+})
 
+/***** Deliverable 2 *****/
+const button = document.querySelector('.like-button')
+const pLike = document.querySelector('.likes')
 
+button.addEventListener('click', function(){
+  const likes = parseInt(pLike.textContent)
+  pLike.textContent = likes + 1 + " Likes"
+})
```

Comment:
The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. Nice work!